core/header.cpp

```diff
@@ -169,6 +169,15 @@ namespace MR
       if (App::project_version)
         H.keyval()["project_version"] = App::project_version;
 
+      std::string cmd = App::argv[0];
+      for (int n = 1; n < App::argc; ++n)
+        cmd += std::string(" \"") + App::argv[n] + "\"";
```

Comment:
@jdtournier Is it necessary to add double-quotes to every entry in `argv`? Could it not be reserved only for entries that contain whitespace?